events/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from django.contrib.auth.models import User
+from users.models import CustomUser
+# Create your models here.
+
+default_user = CustomUser.objects.get(email='ecell@gmail.com')
```

Comment:
why this?

Reply:
This is not the latest version, outdated file